Assets/HighScoreScript.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class HighScoreScript : MonoBehaviour
```

Comment:
Naming a class `*Script`?  I'm thinking more about something like `ScoresManager` or `HighScoresManager`.

Reply:
Change to HighScoresManager